File: MADVCamera/CameraManager/ambaresponse/AMBAGetWiFiStatusResponse.m
//
//  AMBAGetWiFiStatusResponse.m
//  Madv360_v1
//
//  Created by 张巧隔 on 16/9/6.
//  Copyright © 2016年 Cyllenge. All rights reserved.
//

#import "AMBAGetWiFiStatusResponse.h"

@implementation AMBAGetWiFiStatusResponse

- (NSString*) MAC {
    NSDictionary* dict = (NSDictionary*) self.param;
    return dict[@"MAC"];
}

- (NSString*) SSID {
    NSDictionary* dict = (NSDictionary*) self.param;
    return dict[@"SSID"];
}

@end

File: MADVCamera/CameraManager/ambaresponse/AMBAGetWiFiStatusResponse.java
package com.madv360.madv.connection.ambaresponse;

import com.google.gson.annotations.Expose;
import com.madv360.madv.connection.AMBAResponse;

import java.util.Map;

/**
 * Created by qiudong on 16/6/16.
 */
public class AMBAGetWiFiStatusResponse extends AMBAResponse {
    public String MAC() {
        Map resultMap = (Map)param;
        return (String) resultMap.get("MAC");
    }

    public String SSID() {
        Map resultMap = (Map)param;
        return (String) resultMap.get("SSID");
    }
}
